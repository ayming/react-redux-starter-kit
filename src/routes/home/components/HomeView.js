import React, { PureComponent } from 'react'

import Link from '../../../components/Link'
import s from './HomeView.scss'

class HomeView extends PureComponent {
  render() {
    return (
      <div className={s.wrapper}>
        <p>
          To get started, edit{' '}
          <code>src/routes/home/components/HomeView.js</code> and save to
          reload.
        </p>
        <Link to="/about">ABOUT</Link> <Link to="/counter">COUNTER</Link>
      </div>
    )
  }
}

export default HomeView
