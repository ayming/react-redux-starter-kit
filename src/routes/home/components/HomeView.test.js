/* eslint-env jest */
import React from 'react'
import renderer from 'react-test-renderer'

import HomeView from './HomeView'

describe('HomeView', () => {
  test('renders correctly', () => {
    const wrapper = renderer.create(<HomeView />).toJSON()

    expect(wrapper).toMatchSnapshot()
  })
})
