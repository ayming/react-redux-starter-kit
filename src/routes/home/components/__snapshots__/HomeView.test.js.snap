// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeView renders correctly 1`] = `
<div
  className="wrapper"
>
  <p>
    To get started, edit
     
    <code>
      src/routes/home/components/HomeView.js
    </code>
     and save to reload.
  </p>
  <a
    href="/about"
    method="push"
    onClick={[Function]}
  >
    ABOUT
  </a>
   
  <a
    href="/counter"
    method="push"
    onClick={[Function]}
  >
    COUNTER
  </a>
</div>
`;
